### PR TITLE
Remove broken links to unblock CI

### PR DESCRIPTION
**Pull request by roberthbailey**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug

/kind cleanup

> /kind documentation
> /kind feature
> /kind hotfix

**What this PR does / Why we need it**: Remove broken links to unblock CI

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by roberthbailey** [comment]
The link checker seems to have found a few other links to complain about:

```
Html Test: Attempt 20
htmltest started at 07:24:38 on /tmp/website
========================================================================
site/docs/installation/creating-cluster/minikube/index.html
  Non-OK status: 403 --- site/docs/installation/creating-cluster/minikube/index.html --> https://blog.thepolyglotprogrammer.com/setting-up-kubernetes-on-wsl-to-work-with-minikube-on-windows-10-90dac3c72fa1
site/docs/installation/creating-cluster/eks/index.html
  Non-OK status: 403 --- site/docs/installation/creating-cluster/eks/index.html --> https://itnext.io/kubernetes-is-hard-why-eks-makes-it-easier-for-network-and-security-architects-ea6d8b2ca965
========================================================================
✘✘✘ failed in 894.757052ms
```

File: site/content/en/blog/releases/1.12.0-rc.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Check the <a href="https://github.com/googleforgames/agones/tree/release-1.12.0-
 - Move ContainerPortAllocation to Stable [\#1961](https://github.com/googleforgames/agones/pull/1961) ([markmandel](https://github.com/markmandel))
 - CRD OpenAPI Spec for ObjectMeta & PodTemplateSpec [\#1956](https://github.com/googleforgames/agones/pull/1956) ([markmandel](https://github.com/markmandel))
 - Add a "why" section for the Allocator Service documentation [\#1953](https://github.com/googleforgames/agones/pull/1953) ([markmandel](https://github.com/markmandel))
-- Add nodeSelector property to Agones helm chart for Allocator [\#1946](https://github.com/googleforgames/agones/pull/1946) ([josephbmanley](https://github.com/josephbmanley))
+- Add nodeSelector property to Agones helm chart for Allocator [\#1946](https://github.com/googleforgames/agones/pull/1946) (josephbmanley)
```
Comment (pooneh-m) [inline site/content/en/blog/releases/1.12.0-rc.md:34]: remove (josephbmanley)?

File: site/content/en/blog/releases/1.1.0-rc.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Check the <a href="https://github.com/googleforgames/agones/tree/release-1.1.0-r
 **Security fixes:**
 
 - Ran `npm audit fix` to update package dependencies. [\#1097](https://github.com/googleforgames/agones/pull/1097) ([roberthbailey](https://github.com/roberthbailey))
-- Bump eslint-utils from 1.4.0 to 1.4.2 in /sdks/nodejs [\#1014](https://github.com/googleforgames/agones/pull/1014) ([dependabot[bot]](https://github.com/apps/dependabot))
+- Bump eslint-utils from 1.4.0 to 1.4.2 in /sdks/nodejs [\#1014](https://github.com/googleforgames/agones/pull/1014) (dependabot[bot])
```
Comment (pooneh-m) [inline site/content/en/blog/releases/1.1.0-rc.md:46]: Please remove (dependabot[bot]) as well.

**Review by pooneh-m — APPROVED** [review]
Approving to unblock the CL. Please see my comments.

**Pull request merged**